test/Npgsql.Tests/Types/JsonPathTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Threading.Tasks;
+using NpgsqlTypes;
+using NUnit.Framework;
+using static Npgsql.Tests.TestUtil;
+
+namespace Npgsql.Tests.Types
+{
+    public class JsonPathTests : MultiplexingTestBase
+    {
+        public JsonPathTests(MultiplexingMode multiplexingMode)
+            : base(multiplexingMode) { }
+
+        static readonly object[] ReadWriteCases = new[]
+        {
+            new object[] { "'$'::text", "$" },
+            new object[] { "'$\"varname\"'::text", "$\"varname\"" },
+        };
+
+        [Test]
+        [TestCaseSource(nameof(ReadWriteCases))]
+        public async Task Read(string query, string expected)
```

Comment:
I'm probably missing something, but how does this test jsonpath? It looks like it's just reading a `text`.

Reply:
Whoops, will fix. The cast to text was introduced because `jsonpath` lacks the equality operator.